libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -17,7 +17,7 @@ import { TOKENS_LISTS_UPDATER_INTERVAL, getFulfilledResults, getIsTimeToUpdate }
 
 const { atom: lastUpdateTimeAtom, updateAtom: updateLastUpdateTimeAtom } = atomWithPartialUpdate(
   atomWithStorage<Record<SupportedChainId, number>>(
-    'tokens:lastUpdateTimeAtom:v0',
+    'tokens:lastUpdateTimeAtom:v1',
```

Comment:
Why does this need to be updated?

Reply:
We update token lists sources only once in 6 hours.
If we don't change the atom version, the new list will appear only in 6 hours

Reply:
<img width="858" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/3e1dd0e1-f363-41f6-bc86-34dff92550aa">

Reply:
thanks for the response